_posts/2015-03-14-rqlite---building-SQLite-clusters.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: rqlite - building SQLite clusters
+date: 2015-03-14 00:00:00
+description:
+author: oscillatingworks
+hn-discussion:
+tags:
+- raft
+- sqlite
+- iot
+- paxos
+- distributed consensus
+---
+
+Today our compass lead us into direction of [rqlite](https://github.com/otoolep/rqlite). It a is small, comprehensive, [Raft](https://raftconsensus.github.io/) based clustering middleware for [SQLite](https://sqlite.org/) databases. Someone could ask who the hell would need SQLite cluster but first things first.
```

Comment:
I think it's better if you break down this in multiple lines, easier to read on text editor. Same for other paragraphs in the post.